.shared/.post-create

```diff
@@ -1,13 +1,11 @@
-To begin development, install Docker:
+To begin development, install the Temporal CLI:
 
-  Mac: {cyan brew cask install docker; open} /Applications/Docker.app
-  Other: https://www.docker.com/products/docker-desktop
+  Mac: {cyan brew install temporal}
+  Other: Download and extract the latest release from https://github.com/temporalio/cli/releases/latest
 
 Start Temporal Server:
 
-  {cyan git clone} https://github.com/temporalio/docker-compose.git temporal-server
-  {cyan cd} temporal-server/
-  {cyan docker-compose up}
+  {cyan temporal server start-dev}
 
 Use Node version 14+:
```

Comment:
While you're at this... Could you please update this to 14.18? Might even recommend 16+